addins/Cake.Jira.yml

```diff
@@ -0,0 +1,10 @@
+Name: Cake.Jira
+NuGet: Cake.Jira
+Assemblies:
+- "/**/Cake.Jira.dll"
+Repository: https://github.com/Ninglin/Cake.Jira
+Author: Hugo Tavares
+Description: ""
```

Comment:
Can you please add a description for the addin

Reply:
I swear I wrote one 😂  must have gotten lost.